packages/mesos-modules/windows.buildinfo.json

```diff
@@ -0,0 +1,24 @@
+{
+  "sources": {
+    "openssl": {
+      "kind": "url",
+      "url": "https://slproweb.com/download/Win64OpenSSL-1_1_1d.exe",
+      "sha1": "df9c40e9e82edf693c15302cbc8e8a057857a13c"
+    },
+    "mesos": {
+      "kind": "git",
+      "git": "https://github.com/apache/mesos",
+      "ref": "54227a33b68d83b97ef1a7c14283351d45322317",
```

Comment:
This is another place where we need to change version when we bump mesos

Reply:
right... similar place in enterprise modules